From b8f37d10741f52549b8b4cb27f3fe60e672e0a08 Mon Sep 17 00:00:00 2001
From: Jan Schmidt <jan@centricular.com>
Date: Sat, 8 Apr 2017 21:28:58 +1000
Subject: [PATCH] gl: Remove duplicate typedef of GstGLFramebufferClass

The GstGLFramebufferClass struct is typedeffed in
gstgl_fwd.h, and having a duplicate elsewhere is
breaking the cerbero build on my OSX machine,
even though it seems to be working in CI.
---
 gst-libs/gst/gl/gstglframebuffer.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gst-libs/gst/gl/gstglframebuffer.h b/gst-libs/gst/gl/gstglframebuffer.h
index 0d0078a..297a71e 100644
--- a/gst-libs/gst/gl/gstglframebuffer.h
+++ b/gst-libs/gst/gl/gstglframebuffer.h
@@ -35,7 +35,6 @@ GType gst_gl_framebuffer_get_type (void);
 #define GST_IS_GL_FRAMEBUFFER_CLASS(klass)  (G_TYPE_CHECK_CLASS_TYPE((klass), GST_TYPE_GL_FRAMEBUFFER))
 #define GST_GL_FRAMEBUFFER_CAST(obj)        ((GstGLFramebuffer*)(obj))
 
-typedef struct _GstGLFramebufferClass GstGLFramebufferClass;
 typedef struct _GstGLFramebufferPrivate GstGLFramebufferPrivate;
 
 /**
-- 
2.9.3

